packages/apps/plugins/plugin-splitview/src/components/SplitView.tsx

```diff
@@ -45,6 +45,7 @@ export const SplitView = () => {
         {/* Right Complementary sidebar. */}
         {complementarySidebarOpen !== null && (
           <Main.ComplementarySidebar classNames='overflow-hidden'>
+            {/* TODO(burdon): name vs. role? */}
```

Comment:
atm name is the name of the slot vs role which is passed to the surface, I was also wondering if these could be the same thing. It was only in the past couple weeks I think that we got to the point where we had surfaces with both name & role but so far I think they've all been the same.